pkg/apis/certificates/validation/validation.go

```diff
@@ -50,8 +50,14 @@ func ValidateCertificateSigningRequest(csr *certificates.CertificateSigningReque
 	isNamespaced := false
 	allErrs := apivalidation.ValidateObjectMeta(&csr.ObjectMeta, isNamespaced, ValidateCertificateRequestName, field.NewPath("metadata"))
 	err := validateCSR(csr)
+
+	specPath := field.NewPath("spec")
+
 	if err != nil {
-		allErrs = append(allErrs, field.Invalid(field.NewPath("request"), csr.Spec.Request, fmt.Sprintf("%v", err)))
+		allErrs = append(allErrs, field.Invalid(specPath.Child("request"), csr.Spec.Request, fmt.Sprintf("%v", err)))
+	}
+	if csr.Spec.Usages == nil {
```

Comment:
with defaulting, should never be nil, and clients that don't specify should never be empty. I was thinking a zero-length check here should make it invalid